Controllers/param.js

```diff
@@ -0,0 +1,89 @@
+const Cart = require('../Database/cart');
+const Product = require('../Database/product');
+const Wishlist = require('../Database/wishlist');
+const Address = require('../Database/address');
+var jwt = require('jsonwebtoken');
+
+const isAuthorized = (req, res, next) => {
+	const token = req.headers.authorization;
+	try {
+		const decoded = jwt.verify(token, process.env.SECRET_KEY);
```

Comment:
If an invalid token is passed, would that appropriate error be handled within the catch block?
I am not sure if this logic needs an implmentation.